packages/i18n/src/locales/en/user.json

```diff
@@ -1,4 +1,30 @@
 {
+    "changePassword": {
+    "form": {
+      "actions": {
+        "submit": "Change password"
+      },
+      "oldPassword": {
+        "label": "Old password"
+      },
+      "newPassword": {
+        "label": "New password"
+      },
+      "confirmPassword": {
+        "label": "Confirm password is required"
+      }
+    },
+    "messages": {
+      "success": "Change password successfully",
```

Comment:
@shailesh896 I think this should be "Changed" instead of "Change". Or "Your password was successfully changed." might be better.